File: 100-Days-Series/Day 96/SumOfNegativeNumbers.java

class SumOfNegativeNumbers{
    // Method to find the sum of negative numbers in the array.
    public static int getSumOfNegativeNumbers(int arr[]){
        int sum=0;
        for(int i=0;i<arr.length;i++){
            if(arr[i]<0){
                sum+=arr[i];
            }
        }
        return sum;
    }
    // Main Method 
    public static void main(String [] args){
        // Let's Start Here
        int arr[]={5,4,-1,3,-7,-6};
        int result=getSumOfNegativeNumbers(arr);
        System.out.println("Sum of Negative Numbers : "+result);
    }
}

File: 100-Days-Series/Day 96/SumOfNegativeNumbers.c
#include<stdio.h>
// Function to find the sum of negative numbers in the array.
int getSumOfNegativeNumbers(int arr[],int size){
    int sum=0,i=0;
    while(i!=size){
        if(arr[i]<0){
            sum+=arr[i];
        }
        i++;
    }
    return sum;
}
// Let's Start Here
void main(){
    int arr[]={-2, 3, 4, -1, 5};
    int size=sizeof(arr)/sizeof(arr[0]);
    int result=getSumOfNegativeNumbers(arr,size);
    printf("\nSum of Negative Numbers : %d",result);
}

